DynamicData.Tests/DynamicData.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net452;netcoreapp1.1;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;netcoreapp1.1</TargetFrameworks>

```

Comment:
As per you comment on the PR, I suggest we include 461 in the test. 

Also the same should be applied to the corresponding section in DynamicData.csproj

```xml
 <PropertyGroup Condition="'$(TargetFramework)'=='net452'">
    <DefineConstants>P_LINQ</DefineConstants>
  </PropertyGroup>
```
Will need to change to match 461